ee/billing/quota_limiting.py

```diff
@@ -188,23 +189,48 @@ def update_all_org_billing_quotas(dry_run: bool = False) -> Dict[str, Dict[str,
                 org.save(update_fields=["usage"])
 
             for field in ["events", "recordings"]:
-                rate_limited_until = org_quota_limited_until(org, QuotaResource(field))
+                quota_limited_until = org_quota_limited_until(org, QuotaResource(field))
 
-                if rate_limited_until:
+                if quota_limited_until:
                     # TODO: Set this rate limit to the end of the billing period
-                    rate_limited_orgs[field][org_id] = rate_limited_until
+                    quota_limited_orgs[field][org_id] = quota_limited_until
 
-    rate_limited_teams: Dict[str, Dict[str, int]] = {"events": {}, "recordings": {}}
+    # Get the current quota limits so we can track to poshog if it changes
+    orgs_with_changes = set()
+    previously_quota_limited_team_tokens: Dict[str, Dict[str, int]] = {"events": {}, "recordings": {}}
+
+    for field in quota_limited_orgs:
+        previously_quota_limited_team_tokens[field] = list_limited_team_tokens(QuotaResource(field))
+
+    quota_limited_teams: Dict[str, Dict[str, int]] = {"events": {}, "recordings": {}}
 
     # Convert the org ids to team tokens
     for team in teams:
-        for field in rate_limited_orgs:
-            # TODO: Check for specific field on organization to force quota limits on
-            if str(team.organization.id) in rate_limited_orgs[field]:
-                rate_limited_teams[field][team.api_token] = rate_limited_orgs[field][str(team.organization.id)]
+        for field in quota_limited_orgs:
+            org_id = str(team.organization.id)
+            if org_id in quota_limited_orgs[field]:
+                quota_limited_teams[field][team.api_token] = quota_limited_orgs[field][org_id]
+
+                # If the team was not previously quota limited, we add it to the list of orgs that were added
+                if team.api_token not in previously_quota_limited_team_tokens[field]:
+                    orgs_with_changes.add(org_id)
+            else:
+                # If the team was previously quota limited, we add it to the list of orgs that were removed
+                if team.api_token in previously_quota_limited_team_tokens[field]:
+                    orgs_with_changes.add(org_id)
+
+    for org_id in orgs_with_changes:
+        properties = {
+            "quota_limited_events": quota_limited_orgs["events"].get(org_id, None),
+            "quota_limited_recordings": quota_limited_orgs["events"].get(org_id, None),
+        }
+
+        report_organization_action(
```

Comment:
Could we split this in `organization added to quota limit` and `organization removed to quota limit`? Or did CS requested this implementation because of Pocus?

Reply:
We _could_ but it means way more events being tracked. Isn't this what Actions exist for 😅

Reply:
ok ok ok!